docker-compose.yml

```diff
@@ -68,6 +68,8 @@ services:
     volumes:
       - ./:/www
       - /var/run/docker.sock:/var/run/docker.sock
+      # For dev purposes this simulates npm-link, since symbolic links on the mac won't work in the docker container
+      # - ../aries-controller/dist:/www/node_modules/aries-controller
```

Comment:
what's the intention with these added commented out lines?

Reply:
So I have a comment there - this makes it easy for a developer to locally test changes to aries-controller (which is what we did in this PR). Leaving the commented out code around so developers can easily uncomment when they need to test.